libraries/Filter/SlewLimiter.cpp

```diff
@@ -26,21 +26,56 @@
  */
 #include "SlewLimiter.h"
 
-SlewLimiter::SlewLimiter(const float &_slew_rate_max, const float &_slew_rate_tau) :
+SlewLimiter::SlewLimiter(float _slew_rate_max, float _slew_rate_tau) :
     slew_rate_max(_slew_rate_max),
     slew_rate_tau(_slew_rate_tau)
 {
     slew_filter.set_cutoff_frequency(DERIVATIVE_CUTOFF_FREQ);
     slew_filter.reset(0.0);
 }
 
+/*
+  update max slew rate
+ */
+void SlewLimiter::set_max_slew_rate(float _slew_rate_max)
+{
+    _new_slew_rate_max = _slew_rate_max;
+    _reset_filter = true;
+}
+
+/*
+  update max slew rate
+ */
+void SlewLimiter::reset_filter()
```

Comment:
I'm not sure "reset_filter" is a good name because it sounds like it will actually reset the filter but it may not unless the _reset_filter boolean is true.  A comment which somehow clarifies that it might not actually reset the filter could be good.

Reply:
Changed the name